### DISCUSSION TITLE
Icon request: Poker chip icon

### DISCUSSION THREAD

**Discussion by namezero** [FortAwesome/Font-Awesome#21128] [Icon Requests]
Poker chip icon, like coin

**Comment by efosao** [comment]
+1

**Comment by tagliala** [comment]
+1 for both.

gps is a dup of #933 ... as you know :)

renamed as icon poker chip

**Comment by bluntspoon** [comment]
+1

**Comment by ariscapellos** [comment]
+1

**Comment by ghost** [comment]
+1

**Comment by jcolvin1** [comment]
+1

**Comment by j3ko** [comment]
+1

**Comment by uducharm** [comment]
+1

**Comment by dshchupak** [comment]
+1

**Comment by karsumit94** [comment]
+1

**Comment by gayan85** [comment]
+1

**Comment by SkyBadPT** [comment]
+1

**Comment by aleksays** [comment]
+1

**Comment by gokenpei** [comment]
+1

**Comment by cozyartz** [comment]
+1

**Comment by rexatgithub** [comment]
+1

**Comment by karsumit94** [comment]
+1

**Comment by Nikoms** [comment]
+1

**Comment by fanhaipeng0403** [comment]
+1

**Comment by tapastwain** [comment]
+1

**Comment by BonBonSlick** [comment]
+1

**Comment by ghost** [comment]
+1 casino/poker chip.

### What can it be used for?

Used to represent 
* Gambling
* Casinos
* Poker
* Las Vegas
* Macau

### Example images

![7214_3](https://user-images.githubusercontent.com/7556992/40873748-7f771a86-6690-11e8-8562-9d59dbd21c90.jpg)
![117762-200](https://user-images.githubusercontent.com/7556992/40873749-7fae5492-6690-11e8-8c3e-750593a1f49d.png)
![casino_chip_by_fernandesvincent-d4qkut7](https://user-images.githubusercontent.com/7556992/40873750-8065e8e6-6690-11e8-9417-69fd6e759c9b.png)
![casino_chip svg](https://user-images.githubusercontent.com/7556992/40873751-80ac10be-6690-11e8-8e53-ee367daff5f0.png)
![gambling-chip](https://user-images.githubusercontent.com/7556992/40873752-80e2a782-6690-11e8-97f4-a2347e397afc.jpg)
![poker-chip-512](https://user-images.githubusercontent.com/7556992/40873753-8122b214-6690-11e8-8e80-179a9b91507c.png)

**Comment by DungeoN33** [comment]
+1

**Comment by ojako** [comment]
+1

**Comment by kireetig** [comment]
+1

**Comment by sensibleworld** [comment]
This is a quick update to let folks know that we’re refining our icon request workflow behind the scenes, and as part of that we’re going through all open requests and filing them into several new buckets to be addressed.

In this case we’ve added this issue to our new list to be created! We can’t promise any firm timelines right now, but this will help us keep track of it.

We’ll update this issue when the icon is released into the wild 🙂

**Comment by sensibleworld** [comment]
Added in 7.2: https://fontawesome.com/icons/poker-chip?f=classic&s=solid

**Comment by uducharm** [comment]
Thanks
